packages/pluggableWidgets/image-viewer-web/src/ImageViewer.editorConfig.ts

```diff
@@ -36,6 +36,10 @@ export function getProperties(
         hidePropertiesIn(defaultProperties, values, ["iconSize"]);
     }
 
+    if (values.onClickType !== "action") {
+        hidePropertiesIn(defaultProperties, values, ["onClick"]);
```

Comment:
You don't need an array here, can be simply `hidePropertyIn(defaultProperties, values, "onClick")`